sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select "categories"."name" as "category",
+        count("actorId")
```

Comment:
Not a big deal but I would alias the count so when you are looking at the output you know what you counted.